docs/topics/Rate_Limits.md

```diff
@@ -75,85 +75,4 @@ Note that the normal rate-limiting headers will be sent in this response. The ra
   "retry_after": 6457,
```

Comment:
we need to update these examples.

retry-after is now in seconds, not ms

Reply:
Is `64.57` valid since it can be a float

Reply:
`64.57` and `65` would be the header